### PR TITLE
Update templated file

### PR DESCRIPTION
**Pull request by tekktrik**

### CODE REVIEW

File: circuitpython_library_pypi_stats.md [hunk 1]
```diff
@@ -313,3 +312,12 @@
 | Adafruit CircuitPython BLE Beacon (adafruit-circuitpython-ble-beacon) | 113 downloads |
 | Adafruit CircuitPython PCF8575 (adafruit-circuitpython-pcf8575) | 62 downloads |
 | Adafruit CircuitPython ENS160 (adafruit-circuitpython-ens160) | 59 downloads |
+
+## Blinka
```
Comment (kattni) [inline circuitpython_library_pypi_stats.md:316]: This should be a 3-header (`###`) to match the header above the library table above.

File: circuitpython_library_pypi_stats.md [hunk 2]
```diff
@@ -1,10 +1,9 @@
 # Adafruit CircuitPython Library Download Stats
-![Blinka With Tux](https://cdn-learn.adafruit.com/assets/assets/000/089/982/medium640/raspberry_pi_main_image.jpg)  
+![Blinka With Tux](https://raw.githubusercontent.com/tekktrik/adabot/dev/update-file-bundle/assets/BlinkaTux.jpg)
```
Comment (kattni) [inline circuitpython_library_pypi_stats.md:2]: Is this temporary while you await the updated Blinkas?

**Review by kattni — CHANGES_REQUESTED** [review]
Thanks for the update! One change and one question.

File: circuitpython_library_pypi_stats.md (hunk 2 above)
Comment (tekktrik) [reply circuitpython_library_pypi_stats.md:2]: Ah, no, good catch, it should be `adafruit`.

**Review by kattni — APPROVED** [review]
Excellent, thank you!

**Pull request merged**